packages/sdk/tests/utils/mockAllForProcessProtectedData.ts

```diff
@@ -0,0 +1,69 @@
+import { jest } from '@jest/globals';
```

Comment:
Complex file that will mock all necessary `iexec-sdk` methods for a `processProtectedData()` call.